plugins/woocommerce/src/Internal/EmailEditor/PersonalizationTags/OrderTagsProvider.php

```diff
@@ -0,0 +1,161 @@
+					$date_created = $context['order']->get_date_created();
+					if ( ! $date_created ) {
+						return '';
+					}
+					return wc_format_datetime( $date_created, $format );
+				},
+				array(
+					'format' => wc_date_format(),
+				),
+			)
+		);
+
+		$registry->register(
+			new Personalization_Tag(
+				__( 'Order Items', 'woocommerce' ),
+				'woocommerce/order-items',
+				__( 'Order', 'woocommerce' ),
+				function ( array $context ): string {
+					if ( ! isset( $context['order'] ) ) {
+						return '';
+					}
+					$items = array();
+					foreach ( $context['order']->get_items() as $item ) {
+						$items[] = $item->get_name();
+					}
+					return implode( ', ', $items );
+				},
+			)
+		);
+
+		$registry->register(
+			new Personalization_Tag(
+				__( 'Order Subtotal', 'woocommerce' ),
+				'woocommerce/order-subtotal',
+				__( 'Order', 'woocommerce' ),
+				function ( array $context ): string {
+					if ( ! isset( $context['order'] ) ) {
+						return '';
+					}
+					return (string) $context['order']->get_subtotal();
+				},
+			)
+		);
+
+		$registry->register(
+			new Personalization_Tag(
+				__( 'Order Tax', 'woocommerce' ),
+				'woocommerce/order-tax',
+				__( 'Order', 'woocommerce' ),
+				function ( array $context ): string {
+					if ( ! isset( $context['order'] ) ) {
+						return '';
+					}
+					return (string) $context['order']->get_total_tax();
+				},
+			)
+		);
+
+		$registry->register(
+			new Personalization_Tag(
+				__( 'Order Total', 'woocommerce' ),
+				'woocommerce/order-total',
+				__( 'Order', 'woocommerce' ),
+				function ( array $context ): string {
+					if ( ! isset( $context['order'] ) ) {
+						return '';
+					}
+					return (string) $context['order']->get_total();
+				},
+			)
+		);
+
+		$registry->register(
+			new Personalization_Tag(
+				__( 'Payment Method', 'woocommerce' ),
+				'woocommerce/order-payment-method',
+				__( 'Order', 'woocommerce' ),
+				function ( array $context ): string {
+					if ( ! isset( $context['order'] ) ) {
+						return '';
+					}
+					return $context['order']->get_payment_method_title();
+				},
+			)
+		);
+
+		$registry->register(
+			new Personalization_Tag(
+				__( 'Payment URL', 'woocommerce' ),
+				'woocommerce/order-payment-url',
+				__( 'Order', 'woocommerce' ),
+				function ( array $context ): string {
+					if ( ! isset( $context['order'] ) ) {
+						return '';
+					}
+					return $context['order']->get_checkout_payment_url();
+				},
+			)
+		);
+
+		$registry->register(
+			new Personalization_Tag(
+				__( 'Admin Order Note', 'woocommerce' ),
+				'woocommerce/admin-order-note',
+				__( 'Store', 'woocommerce' ),
+				function ( array $context ): string {
+					if ( isset( $context['wc_email'], $context['wc_email']->customer_note ) ) {
+						return wptexturize( $context['wc_email']->customer_note );
+					}
+					return '';
+				},
+			)
+		);
```

Comment:
Let's move this to the StoreTagsProvider for now. We plan to remove the tag when the custom block is created

Reply:
Oops, that was not intentional. Fixed it.